src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -79,6 +79,13 @@ protected override void OnCreateControl()
 			}
 		}
 
+		/// <summary>
+		/// Path to the start page
+		/// </summary>
+		[Category("Behavior")]
+		[Description(@"Path for initial navigation when the application starts.")]
```

Comment:
Similar to the doc comment change, perhaps something like this?

```suggestion
		[Description(@"Path for initial Blazor navigation when the Blazor component is finished loading.")]
```